Source/Krypton Components/Krypton.Toolkit/Controls Visuals/Notification Toast/Basic/VisualBasicToastForm.cs

```diff
@@ -0,0 +1,302 @@
+ *
+ */
+#endregion
+
+using Timer = System.Windows.Forms.Timer;
+
+namespace Krypton.Toolkit
+{
+    public partial class VisualBasicToastForm : KryptonForm
+    {
+        #region Instance Fields
+
+        private readonly bool _showActionButton;
+
+        private readonly bool _showCloseButton;
+
+        private Font? _notificationContentFont;
+
+        private Font? _notificationTitleFont;
+
+        private int _time;
+
+        private int? _countDownSeconds;
+
+        private readonly string _notificationContentText;
+
+        private readonly string? _notificationTitleText;
+
+        private readonly string? _notificationSoundPath;
+
+        private readonly string? _processPath;
+
+        private object? _optionalParameters;
+
+        private Timer _timer;
+
+        private readonly Image? _customIcon;
+
+        private SoundPlayer? _soundPlayer;
+
+        private Stream? _customSoundStream;
+
+        private SystemSound? _systemSound;
+
+        private KryptonCommand? _actionButtonCommand;
+
+        private KryptonToastNotificationIcon? _toastNotificationIcon;
+
+        private InputControlStyle? _inputControlStyle;
+
+        private KryptonToastNotificationActionType? _actionType;
+
+        private KryptonToastNotificationActionButton? _actionButton;
+
+        #endregion
+
+        #region Identity
+
+        /// <summary>Initializes a new instance of the <see cref="VisualBasicToastForm" /> class.</summary>
+        /// <param name="notificationContentText">The notification content text.</param>
+        /// <param name="notificationTitleText">The notification title text.</param>
+        /// <param name="notificationSoundPath">The notification sound path.</param>
+        /// <param name="notificationTitleFont">The notification title font.</param>
+        /// <param name="notificationContentFont">The notification content font.</param>
+        /// <param name="countDownSeconds">The count-down seconds.</param>
+        /// <param name="processPath">The process path.</param>
+        /// <param name="optionalParameters">The optional parameters.</param>
+        /// <param name="showActionButton">The show action button.</param>
+        /// <param name="showCloseButton">The show close button.</param>
+        /// <param name="customSoundStream">The custom sound stream.</param>
+        /// <param name="systemSound">The system sound.</param>
+        /// <param name="actionButtonCommand">The action button command.</param>
+        /// <param name="toastNotificationIcon">The toast notification icon.</param>
+        /// <param name="customIcon">The custom icon.</param>
+        /// <param name="inputControlStyle">The input control style.</param>
+        /// <param name="actionType">Type of the action.</param>
+        /// <param name="actionButton">The action button.</param>
+        public VisualBasicToastForm(string notificationContentText, string? notificationTitleText, 
```

Comment:
Why is this not a structure parameter from the "get go"

Reply:
Going to test this with fields for now, then implement the existing structure at a later date, until I'm sure that everything is functioning properly.